rotkehlchen/chain/aggregator.py

```diff
@@ -615,10 +615,10 @@ def sync_bitcoin_accounts_with_db(
         """
         db_btc_accounts = getattr(
             self.database.get_blockchain_accounts(cursor),
-            blockchain.value.lower(),
+            blockchain.get_key(),
```

Comment:
here using get_key() just to differentiate between external and internal function? As they seem to be the same.

Reply:
Yes to have clear what is being sent to the frontend and what not